// my-library-lib/esm5/lib/my-library-lib.component.js
/**
 * @fileoverview added by tsickle
 * @suppress {checkTypes,extraRequire,missingReturn,unusedPrivateMembers,uselessCode} checked by tsc
 */
import { Component } from '@angular/core';
var MyLibraryLibComponent = /** @class */ (function () {
    function MyLibraryLibComponent() {
        this.nombre = "guillermo de mendoza";
    }
    /**
     * @return {?}
     */
    MyLibraryLibComponent.prototype.ngOnInit = /**
     * @return {?}
     */
    function () {
    };
    MyLibraryLibComponent.decorators = [
        { type: Component, args: [{
                    selector: 'en1-myLibrary-lib',
                    template: "<p class=\"setColor\">yea is working</p>\n",
                    styles: [".setColor{color:red;text-align:center}"]
                }] }
    ];
    /** @nocollapse */
    MyLibraryLibComponent.ctorParameters = function () { return []; };
    return MyLibraryLibComponent;
}());
export { MyLibraryLibComponent };
if (false) {
    /** @type {?} */
    MyLibraryLibComponent.prototype.nombre;
}
